### ISSUE TITLE
Hyperlink in slide heading stopped working

### ISSUE THREAD

**Issue by MartinPacker** [MartinPacker/md2pptx#125]
Following syntax leads to the heading being swallowed:

```
### Some Additional WLM-Related Information From [IBM](http://w3.ibm.com)
```

Conirmed in 3.3.2

**Comment by MartinPacker** [comment]
It looks like:

1. The MultiMarkdown capability to create an id for a heading using `[myref]` is swallowing any following text. Possibly correct - if we restrict this syntax to the end of the line.
1. There never was a capability to code **outbound** hyperlinks in a heading. Certainly not in Markdown. Proposal is to enable this and mark it as a deviation from (actually extension to) Markdown. It would be useful. (I ran into two separate presentations where I could've used it.)

**Comment by MartinPacker** [comment]
1. Fixed with a simple `$` in the regex.
1. Having fixed the bug it turns out links - both internal and external - work just fine.

It remains to document the deviation and then push as 3.3.3.

**Comment by MartinPacker** [comment]
Shipped in [v3.3.3](https://github.com/MartinPacker/md2pptx/releases/tag/v3.3.3).
